features/shop/apply_correct_shipping_fee_on_order.feature

```diff
@@ -10,6 +10,7 @@ Feature: Apply correct shipping fee on order
         And store has a product "PHP T-Shirt" priced at "€100.00"
         And store has "DHL" shipping method with "€10.00" fee
         And store has "FedEx" shipping method with "€30.00" fee
+        And store allows paying offline
```

Comment:
Unfortunately yes. Step ``When I proceed selecting "..." shipping method`` requires default "Offline" payment method, as taxes (for example for shipping) are applied after "payment" checkout step (``sylius.checkout.payment.pre_complete``);
In the future, when we add proper callbacks on order state it will be unnecessary, but know it would force us to deal with listeners priorities ;/